e2e/helper/helper.go

```diff
@@ -16,15 +17,17 @@ import (
 	"github.com/0xPolygon/pbft-consensus"
 )
 
+const trueString = "true"
```

Comment:
hol'up, do we really need this?

Reply:
Just because of annoying linter warnings. :roll_eyes:

Reply:
Although this is non-substantial change, agreed with Momčilo on chat, not to revert this back, because of linter.